packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -106,8 +110,8 @@ export function Autocomplete({
         placeholder={placeholder}
         onFocus={handleInputFocus}
         onBlur={handleInputBlur}
-        name={name}
```

Comment:
The `name` prop is included in the `inputProps` spread object, so we no longer need to explicitly pass it to the child.

Reply:
@darryltec - It looks like the `invalid` prop is in a similar scenario. Any issues if I give it the same treatment here?

Reply:
nope. no issue at all. clean that thang up!

Reply:
Done in e25b098, thanks!